src/cgns_header.h

```diff
@@ -918,6 +1024,7 @@ typedef struct {            /* CGNSBase_t Node          */
     CGNS_ENUMT(DataClass_t) data_class; /* Class of data                        */
     cgns_units *units;      /* Dimensional Units            */
     cgns_equations *equations; /* ptrs to in-mem. copy of FlowEqu.  */
+    cgns_pequations *pequations; /* ptr to in-mem. copy of ParticleEqu. */
```

Comment:
@MicK7, For example, here, I think we might add a version of when it was introduced. This would also be useful for the Doxygen docs.

Reply:
I fully agree we need the version here.